detection.py

```diff
@@ -45,9 +47,10 @@ def draw_contours_mask(binary, contours):
 def get_fill_masks(image):
     # ищем границы
     closed = get_edges(image)
-    # находим контуры
+    # находим контуры листа
     contours, _ = cv2.findContours(closed.copy(), cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
```

Comment:
```suggestion
    contours, _ = cv2.findContours(closed, cv2.RETR_EXTERNAL, cv2.CHAIN_APPROX_SIMPLE)
```
`findContours` не изменяет входное изображение

Reply:
Исправила